parsec/README.md

```diff
@@ -50,6 +63,8 @@ parsec *.c
 The output of `parsec` is a single JSON file, `analysis.json`, created in the current working directory. The format of this file is described herein:
 
 ```json
+# MDE: it is essential to give the meaning of each field, and also the formats of strings.
+# Or, cross-reference to where that information is documented.
```

Comment:
@vikramnitin9 Can you please document these? I think this is work which is also captured by the issue I opened a few weeks ago: https://github.com/vikramnitin9/rust_verification/issues/13